app/pages/ServiceDiscoveryResults/SearchResults/SearchResults.scss

```diff
@@ -18,6 +18,18 @@
   padding-left: 24px;
   border-left: 1px solid #F4F4F4;
   border-bottom: 1px solid #F4F4F4;
+
+  @media screen and (max-width: 1120px) {
+    flex-direction: column;
+  }
+
+  @media screen and (max-width: 600px) {
+    padding-left: 21px;
+    padding-top: 18px;
+    padding-bottom: 18px;
```

Comment:
Is there a reason the padding is asymmetric between left and right? This will cause some things to be uncomfortably close to the right side of the screen.

Reply:
Yeah, shouldn't be there. Was following the figma too close.